www/js/search.js

```diff
@@ -800,6 +821,7 @@ module.exports = {
     };
 
     const currentRows = rows.slice(start, end);
+    let lineIDConflict = false;
```

Comment:
when english explanations are off the down arrow key stop working when on the line **"nanak har gun gai alakh prabh laakheeai"** and on the line **"habhe saak kooraave..."** probable something with the ids
![image](https://user-images.githubusercontent.com/2982784/57713748-e936de00-7627-11e9-8c0c-4ae8f7e2b2a5.png)